packages/apps/contracts/HighRollerApp.sol

```diff
@@ -120,59 +120,39 @@ contract HighRollerApp is CounterfactualApp {
     return abi.encode(nextState);
   }
 
-  function resolve(bytes calldata encodedState, Transfer.Terms calldata terms)
+  function resolve(bytes calldata encodedState)
     external
     pure
-    returns (Transfer.Transaction memory)
+    returns (TwoPartyEthAsLump.Resolution)
   {
     AppState memory appState = abi.decode(encodedState, (AppState));
 
-    uint256[] memory amounts = new uint256[](2);
-    address[] memory to = new address[](2);
-    to[0] = appState.playerAddrs[0];
-    to[1] = appState.playerAddrs[1];
     bytes32 expectedCommitHash = keccak256(
       abi.encodePacked(appState.salt, appState.playerFirstNumber)
     );
     if (expectedCommitHash == appState.commitHash) {
-      amounts = getWinningAmounts(
-        appState.playerFirstNumber, appState.playerSecondNumber, terms.limit
+      return getWinningAmounts(
+        appState.playerFirstNumber, appState.playerSecondNumber
       );
     } else {
-      amounts[0] = 0;
-      amounts[1] = terms.limit;
+      return TwoPartyEthAsLump.Resolution.SEND_TO_ADDR_TWO;
```

Comment:
Why did you change the `else` to send it to address two? It's meant to split it, right?

Reply:
The old code did not split it